src/setupProxy.js

```diff
@@ -1,30 +1,36 @@
 const { createProxyMiddleware } = require('http-proxy-middleware');
 
 module.exports = function(app) {
-  const password = process.env.PROXY_PASSWORD;
-  const username = process.env.PROXY_USERNAME;
+  const env_prefix = process.env.PROXY_PREFIX || 'STAGING';
+  const url = process.env[env_prefix + '_URL'] || 'https://nens.lizard.net/';
```

Comment:
This env_prefix makes things more complicated I think.
I understand it allows users to specify more PROXY_PREFIX variables in their ~./profile file.
But also in order to do that they would need to read and understand this code first

Reply:
They can ignore it, there is a default. It's purely optional. There is also explanation in the README that should be clear enough without having to look at the code. Isn't it? Is there something that can be improved about that?

What *is* daft is that I see that I made 'STAGING' the default prefix, but 'nens.lizard.net' the default domain. That doesn't work and needs to be fixed. Which one should be default, staging or production?

Ideally it should do the right thing with 1 setting, for the api key. And the rest is optional.